### PR TITLE
SEO-175114 .NET MAUI Maps Interlinking

### PR DESCRIPTION
**Pull request by Yvone-Atieno**
Hi @Girija-Rajendran,

|Title|Description|
|--|--|
|Task Category| .NET MAUI Maps|
|Content Task Link/Mail Screenshot| NA|
|UX task| NA|
|Interlinking PR link| Interlinking PR -https://github.com/syncfusion-content/maui-docs/pull/1394|
|Ticket/Task link| https://dev.azure.com/Syncfusion-Infrastructure/Syncfusion%20SEO/_workitems/edit/175114/|
|Excel/SharePoint link| https://syncfusion.sharepoint.com/sites/GH/SEO/SitePages/NET-MAUI-Maps-Money-Keywords.aspx|

| Keyword | Volume | Position | Reason for change | Changes Made |
| ---      | ---       | ---     | ---     | ---     |
|.net maui maps|0-10 |3FT|To improve our FT experience |added CSS classes alignment format, images aligned in zigzag form, added documentation, code example,renamed images & removed unwanted space |
|.net maui maps example|0-10 |3UG,4FT|To improve page experience | demo link added|
|.net maui maps control |0-10 |4FT|To improve our FT experience | improved on page changes|
|.net maui google maps |0-100|6FT |To improve FT ranking to 1st position| added CSS classes alignment format, images aligned in zigzag form, added documentation & Example links, unwanted space removed|
|.net maui maps custom pin|0-10 |5UG| To improve our UG experience |added corresponding documentation links, arranged images in zigzag form|

 

Worked on 16 keywords in the .NET MAUI Maps and am keeping track of all keyword details in the link below:-
https://syncfusion.sharepoint.com/sites/GH/SEO/SitePages/NET-MAUI-Maps-Money-Keywords.aspx

### CODE REVIEW

File: MAUI/Maps/getting-started.md [hunk 1]
```diff
@@ -577,6 +577,4 @@ this.Content = map;
 
 ![.NET MAUI Maps with tile layer](images/getting-started/getting_started_tile_layer.png)
 
-N> 
-* Get the complete getting started sample from [GitHub](https://github.com/SyncfusionExamples/.NET-MAUI-Maps-Getting-Started) link.
-* You can refer to our [.NET MAUI Maps](https://www.syncfusion.com/maui-controls/maui-maps) feature tour page for its groundbreaking feature representations.
\ No newline at end of file
+N> You can refer to our [.NET MAUI Maps](https://www.syncfusion.com/maui-controls/maui-maps) feature tour page for its groundbreaking feature representations. You can also explore our [.NET MAUI Maps](https://github.com/syncfusion/maui-demos/tree/master/MAUI/Maps) example that shows you how to configure the Maps in .NET MAUI.
```
Comment (Sharly-Atieno) [inline MAUI/Maps/getting-started.md:580]: Change as bellow:
[.NET MAUI Maps Example](https://github.com/syncfusion/maui-demos/tree/master/MAUI/Maps) that shows you how to render the Maps in .NET MAUI.

**Review by Sharly-Atieno — CHANGES_REQUESTED** [review]

File: MAUI/Maps/getting-started.md (hunk 1 above)
Comment (Yvone-Atieno) [reply MAUI/Maps/getting-started.md]: changed

File: MAUI/Maps/getting-started.md [hunk 2]
```diff
@@ -577,6 +577,4 @@ this.Content = map;
 
 ![.NET MAUI Maps with tile layer](images/getting-started/getting_started_tile_layer.png)
 
-N> 
-* Get the complete getting started sample from [GitHub](https://github.com/SyncfusionExamples/.NET-MAUI-Maps-Getting-Started) link.
-* You can refer to our [.NET MAUI Maps](https://www.syncfusion.com/maui-controls/maui-maps) feature tour page for its groundbreaking feature representations.
\ No newline at end of file
+N> You can refer to our [.NET MAUI Maps](https://www.syncfusion.com/maui-controls/maui-maps) feature tour page for its groundbreaking feature representations. You can also explore our [.NET MAUI Maps Example](https://github.com/syncfusion/maui-demos/tree/master/MAUI/Maps) that shows you how to render the Maps in .NET MAUI.
```
Comment (Girija-Rajendran) [inline MAUI/Maps/getting-started.md:580]: why did you remove this?
image

**Review by Girija-Rajendran — APPROVED** [review]
